test/torchaudio_unittest/functional/librosa_compatibility_cuda_test.py

```diff
@@ -0,0 +1,12 @@
+from torchaudio_unittest.common_utils import PytorchTestCase, skipIfNoCuda
+from .librosa_compatibility_test_impl import Functional, FunctionalComplex
+
+
+@skipIfNoCuda
+class TestFunctionalCUDA(Functional, PytorchTestCase):
+    device = 'cuda'
+
+
+@skipIfNoCuda
+class TestFunctionalComplexCPU(FunctionalComplex, PytorchTestCase):
```

Comment:
do you mean for this to be `TestFunctionalComplexCUDA` and `device = 'cuda'`?

Reply:
Yes, I forgot to fix it after copy-pasting the code from CPU version. Thanks for catching.